README.md

```diff
@@ -13,13 +13,13 @@ Follow us on Twitter [@6fusion](https://twitter.com/6fusion)
 * cd vmware-collector
 * bundle install
 * Verify `config/{ENVIRONMENT}/mongoid.yml` file
-* Create a “secrets” folder where you create 2 files: uc6 and vsphere (see samples on secrets_example folder)
+* Create a “secrets” folder where you create 2 files: on_prem and vsphere (see samples on secrets_example folder)
```

Comment:
We need to update the readme, as we do not have secret files. We only have the vmwarecollector.yaml
We need to also let the user know that they need to change the env variable where it is set the environment(development,staging,etc)